airbyte-webapp/src/components/ui/ListBox/ListBox.module.scss

```diff
@@ -3,8 +3,21 @@
 
 .button {
   width: 100%;
-  height: 100%;
   cursor: pointer;
+  border: 1px solid colors.$grey-50;
+  background-color: colors.$grey-50;
+  border-radius: 4px;
+  text-align: left;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 14px;
+  line-height: 20px;
+  min-height: 36px;
+
+  &:hover {
+    border: 1px solid colors.$grey-100;
```

Comment:
Since we're just changing the color here, it might be nicer actually just setting` border-color` without setting the width and style again.